clients/java/src/main/java/io/camunda/zeebe/client/api/command/ModifyProcessInstanceCommandStep1.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.command;
+
+import io.camunda.zeebe.client.api.response.ModifyProcessInstanceResponse;
+import java.io.InputStream;
+import java.util.Map;
+
+public interface ModifyProcessInstanceCommandStep1 {
+
+  /**
+   * Create an activate instruction for the given element id. The element will be created within an
```

Comment:
🔧 In all the other method docs you link to the ActivateInstruction, please do so here for consistency

Reply:
😱 I missed one